knowledge-base/pdfprocessing-merge-two-pages-from-different-documents-into-a-single-page.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Merge two document pages into a new single page
+description: This knowledge base article describes how import two PDF documents and merge their first pages into a new single page 
```

Comment:
This knowledge base article describes how to import two PDF documents and merge their first pages into a new single page.